hal/fallback/FallbackScreenPower.cpp

```diff
@@ -27,15 +31,41 @@ void
 SetKeyLightEnabled(bool aEnabled)
 {}
 
+// alternative: use libxrand directly
+// see http://cgit.freedesktop.org/xorg/app/xbacklight/tree/xbacklight.c
```

Comment:
Using libxrandr makes more sense than spawning a process (`xbacklight` that uses libxrandr) each time we want to change the backlight.
The benefit to using xbacklight is that was faster to implement :-)